tasks/es/cnig.py

```diff
@@ -44,6 +44,16 @@ def input_shp(self):
                                 resolution=self.resolution.upper()))
         return os.path.join(self.input().path, path)
 
+class CNIG(TagsTask):
+    def version(self):
+        return 1
+    def tags(self):
+        return [
+            OBSTag(id='CNIG',
```

Comment:
Right now, IDs for tags aren't qualified by type.  This should therefore be `CNIG-source`, or better, `cnig-source` (we should avoid uppercase).